vue-components/src/components/OptionsMenu.vue

```diff
@@ -130,10 +130,14 @@ export default Vue.extend( {
 			const element = this.$refs[ 'menu-items' ] as HTMLElement[];
 
 			if ( this.keyboardHoveredItemIndex !== -1 ) {
-				element[ this.keyboardHoveredItemIndex ].scrollIntoView( {
-					behavior: 'smooth',
-					block: 'end',
-					inline: 'nearest',
+				/**
+				 * This setTimeout shouldn't be needed, but it is a workaround to make scrollIntoView with options
+				 * work in Chrome 88, but that problem likely existed on older Chrome versions as well.
+				 *
+				 * TODO: With newer versions of Chrome, try removing the setTimeout and see if it works.
+				 */
+				setTimeout( () => {
+					element[ this.keyboardHoveredItemIndex ].scrollIntoView( { behavior: 'smooth', block: 'nearest' } );
```

Comment:
Shouldn't it have a timeout number? like 300ms?

Reply:
No, without a number (or with `0`) it is just scheduled run after the current thread has finished. We don't actually want a delay there, so this should be fine.

Reply:
aha I remember now. it's mentioned in https://www.youtube.com/watch?v=8aGhZQkoFbQ